Description: Link to -latomic by default
 This avoids surprises on mips*el/ppc*el
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2019-10-25
Forwarded: not-needed
Bug: https://github.com/nodejs/node/pull/28532
Bug: https://github.com/nodejs/node/issues/30093
--- a/node.gypi
+++ b/node.gypi
@@ -341,6 +341,9 @@
       'libraries!': [
         '-lrt'
       ],
+      'libraries': [
+        '-latomic'
+      ]
     }],
     [ 'OS in "freebsd linux"', {
       'ldflags': [ '-Wl,-z,relro',
